.github/workflows/pytest_apps.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
+        pip install flake8 pytest mock
```

Comment:
Should we add `mock` to `pyproject.toml` as well? Maybe in the future we want to structure `pyproject.toml` a bit for dev mode.

Reply:
Nope. This change is for tests only. `mock` is of no use for the package.

Reply:
I mean we should create a group for the test dependencies in `pyproject.toml` like `[tool.poetry.group.test.dependencies]`.